Zend/tests/gc_049.phpt

```diff
@@ -0,0 +1,68 @@
+--TEST--
+GC 049: Test objects created during GC cycle does not participate in the same cycle
```

Comment:
It is an implementation detail that garbage cycles created in a destructor are not immediately collected. It is also not a desired property (in an ideal world these would be collected immediately), so I don't think we should test that.

Reply:
I have added this test as I want to rely on this behaviour.

> garbage cycles created in a destructor are not immediately collected

In practise, I belive the tested behaviour is the only one wanted. The reason is when a cycle is created, GC is needed. When GC would be possibly rerun within a GC cycle, there will be infine loop. Can we agree?

Reply:
Agreed, it's unlikely to change. We can include this test then